src/Interfaces/SearchHandlerInterface.php

```diff
@@ -3,8 +3,15 @@
 
 namespace LoyaltyCorp\Search\Interfaces;
 
-interface HandlerInterface
+interface SearchHandlerInterface
```

Comment:
This is a BC break which we need to avoid going forwards.

Pagey's team is looking at or experimenting with this package at the moment so you'll need to check with him if this change will cause work for his team, and you'll need to update Subscriptions to handle it.

A way to work around this kind of issue is to keep a file at HandlerInterface.php that contains:

```
/**
 * @deprecated Migrate to SearchHandlerInterface
 */
interface HandlerInterface extends SearchHandlerInterface {}
`

Which means that this change wouldnt break anything and we wouldnt have to bump the semver major